Assets/VRTK/SDK/Daydream/DaydreamControllerLog.cs

```diff
@@ -0,0 +1,169 @@
+﻿using UnityEngine;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+
+public class DaydreamControllerLog : MonoBehaviour
+{
+    public bool consoleLogging;
+    public bool csvLogging;
+    public string csvPath = @"d:\ControllerData.csv";
+
+    private StreamWriter writer;
+
+    void Start()
+    {
+        if (csvLogging)
+        {
+            ControllerCSVStart();
+        }
+    }
+
+    void Update()
```

Comment:
monobehaviour methods should all be protected virtual to allow script extension if required.